resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,132 @@
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "s_c's heart lurches seeing the apprentice so low. Could s_c have done something to make app1 feel this way? Not having any words s_c sits close to app1, wrapping {PRONOUN/p_l/poss} tail around {PRONOUN/app1/object}. s_c purrs softly, reassuringly. Until app1 purrs back, gratefully. ",
+                "exp": 10,
+                "can_have_stat": ["p_l"],
+                "stat_trait": ["insecure","nervous","oblivious","careful","responsible"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "s_c shifts awkwardly at app1's side until {PRONOUN/s_c/subject} can't stand the silence any longer. s_c drops into a crouch and nudges app1 with {PRONOUN/s_c/poss} paw. App1 looks back, confused as s_c keeps bouncing and poking at {PRONOUN/s_c/object} with a goofy expression on {PRONOUN/s_c/poss} face, until app1 gives in. The warrior and apprentice play like kits, and the smile on app1's face says {PRONOUN/app1/subject} {VERB/app1/has/have} forgotten all about {PRONOUN/app1/poss} doubt.",
+                "exp": 10,
+                "can_have_stat": ["p_l"],
+                "stat_trait": ["charismatic","flamboyant","playful","childish","shameless","strange","sneaky"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "s_c tries to think of something to say or do, anything, but {PRONOUN/s_c/subject} {VERB/s_c/fumbles/fumble} as app1 becomes more and more unconsolable. Finally, s_c gives up. Leaving app1 alone with {PRONOUN/app1/poss} thoughts.",
+                "exp": 0,
+                "can_have_stat": ["p_l"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": -10
+                    } ],
+                "weight": 10
+            }
+        ]
     }
+    
 ]
```

Comment:
SQUARE BRACKET ALERT

Reply:
2955's } encloses failure outcome number 1. 2956's ] encloses the failure outcome block as a whole. Then 2957's } encloses the patrol code for this patrol entirely